locales/en/translations.json

```diff
@@ -1,4 +1,33 @@
 {
+    "addGeneralPartnerLegalEntityPage" : {
+        "title" : "Add a General Partner (legal entity)",
```

Comment:
Here and in the Welsh file, 'General Partner' should be all in lower-case. Blame me as I got this wrong on the Person page.

Reply:
updated